src/types.rs

```diff
@@ -458,16 +318,16 @@ pub struct RuntimeConfig {
 	#[serde(default)]
 	#[serde(with = "port_range_format")]
 	pub http_server_port: (u16, u16),
-	/// Seed for IPFS keypair. If not set, or seed is 0, random seed is generated
-	pub ipfs_seed: Option<u64>,
-	/// IPFS service port range (port, range) (default: 37000).
+	/// Seed for Libp2p keypair. If not set, or seed is 0, random seed is generated
+	pub libp2p_seed: Option<u8>,
+	/// Libp2p service port range (port, range) (default: 37000).
 	#[serde(default)]
 	#[serde(with = "port_range_format")]
-	pub ipfs_port: (u16, u16),
-	/// File system path where IPFS service stores data (default: avail_ipfs_node_1)
-	pub ipfs_path: String,
+	pub libp2p_port: (u16, u16),
 	/// RPC endpoint of a full node for proof queries, etc. (default: http://127.0.0.1:9933).
 	pub full_node_rpc: Vec<String>,
+	/// File system path where psk key is stored
+	pub libp2p_psk_path: String,
```

Comment:
This parameter should probably be optional, with the default value of path not set...

Reply:
this is handled through the `get_psk` function
since the return of that one is `Result<Option<String>>`

Reply:
This will lead to partial moves, which occurs because type `std::option::Option<std::string::String>` does not implement the `Copy`.

Best way is to leave it as is, and use empty string